### ISSUE TITLE
In which presentation must the contrast requirements be met? (1.4.3, 1.4.11)

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#2271]
It is obvious that the user is responsible for the resulting contrasts when the user adjusts the colors on a web page (e.g. High Contrast or Dark Mode in the operating system, color adjustment in the browser, adjustment via userstyles etc.).

But what if the web author prevents the adjustment of the colors by querying the user's settings via CSS media queries (`forced-colors, prefers-color-scheme, prefers-contrast`) and then adjusts the colors or prevents the adjustment of the colors? Is the author expected to accept the user's settings (the browser is obliged to do so according to UAAG)? If not, why not?

If yes:
* what content could be non-adjustable (graphics?),
* what minimum contrasts must be respected (for `prefers-contrast`=more maybe 7:1 instead of 4,5:1),
* what maximum contrasts must be kept (with `prefers-contrast`=less)
* what must be light or dark for `prefers-color-scheme`=light/dark?
* is it sufficient to conform with WCAG contrasts SCs only  in one user's display mode? If so, does that have to be the default mode? If it does not have to be the default mode: How must the user be informed that the default mode does not have sufficient contrasts, but another mode (of the browser or operating system) does?

This seems to me to be unresolved at this time because CSS techniques are newer than WCAG. If it is not possible to update to the existing Understandings, I would suggest to consider this from WCAG 2.2 or 2.3. In my opinion it would be absurd if the accessibility features of the browsers and operating systems (regarding high contrast or dark mode etc.) can be undermined by the author.

Related: https://github.com/w3c/wcag/issues/1209

**Comment by fstrr** [comment]
When an author creates styles for these preferences, they're creating page variations so are responsible for ensuring that those variations are conforming. For example: if there were custom styles for `prefers-contrast`, the responsibility is with the author to ensure those styles conform. Where there are styles for a user-controlled preference, for example: `forced-colors: active`, where the user can choose their own colors in something like Windows high-contrast mode, then those are outside the conformance scope.

**Comment by JAWS-test** [comment]
@fstrr I agree, but this leaves most of my questions above unanswered

**Comment by electrickite** [comment]
Possibly related to #2889.
